### PR TITLE
Use abbreviations instead of functions for simple aliases

### PR DESCRIPTION
**Pull request by niklasnatter**
I think it would be nice to use abbreviations instead of functions when possible because it allows the user to see the full command easily.

Closes https://github.com/jhillyerd/plugin-git/issues/97

### CODE REVIEW

**Comment by jhillyerd** [comment]
Can you revert some of the unnecessary readme changes?  It's intentional that less commonly used commands are described, instead of being pasted in directly, I don't want folks to have to dive into the git docs immediately to even understand what something does.

**Comment by niklasnatter** [comment]
> Can you revert some of the unnecessary readme changes? It's intentional that less commonly used commands are described, instead of being pasted in directly, I don't want folks to have to dive into the git docs immediately to even understand what something does.

Yeah sure, no problem. Which ones should be reverted/should stay?

File: README.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ gunwip           # restore work in progress
 | gdca         | `git diff --cached`                                  |
 | gds          | `git diff --stat`                                    |
 | gdsc         | `git diff --stat --cached`                           |
-| gdt          | list changed files                                   |
+| gdt          | `git diff-tree --no-commit-id --name-only -r`        |
```
Comment (jhillyerd) [inline README.md:174]: revert gdt

File: README.md [hunk 2]
```diff
@@ -232,11 +232,11 @@ gunwip           # restore work in progress
...
-| ggp          | push origin _current-branch_                         |
-| ggp!         | `ggp --force-with-lease`                             |
-| gpu          | `ggp --set-upstream`                                 |
-| gpoat        | push all + tags to origin                            |
-| ggpnp        | pull & push origin _current-branch_                  |
+| ggp          | `git push origin (__git.current_branch)`             |
+| ggp!         | `git push origin (__git.current_branch) --force-with-lease`   |
+| gpu          | `git push origin (__git.current_branch) --set-upstream`       |
+| gpoat        | `git push origin --all; and git push origin --tags`  |
+| ggpnp        | `git pull origin (__git.current_branch); and git push origin (__git.current_branch)`    |
```
Comment (jhillyerd) [inline README.md:239]: too long, revert ggpnp

**Review by jhillyerd — CHANGES_REQUESTED** [review]

**Pull request merged**